src/components/TileCatalogNew/TilePagination/TilePagination.jsx

```diff
@@ -86,13 +86,105 @@ const TilePagination = ({ page, numPages, onChange, i18n }) => {
       {pageNumber}
     </button>
   );
+  const getPageSelect = (pageNumber, accumulator) => (
+    <li className="bx--pagination-nav__list-item">
+      <div className="bx--pagination-nav__select">
+        <select
+          className="bx--pagination-nav__page bx--pagination-nav__page--select"
+          data-page-select
+          aria-label="select page number"
+          onChange={evt => onChange(Number(evt.target.value))}
+        >
+          <option value="default" hidden data-page="" />
+          {Array.from({ length: pageNumber }, (v, i) => (
+            <option value={i + accumulator} data-page={i + accumulator}>
+              {i + accumulator}
+            </option>
+          ))}
+        </select>
+        <div className="bx--pagination-nav__select-icon-wrapper">
```

Comment:
Needs carbon prefix variable.
Is the reasoning to use this specific carbon classname to inherit those styles?

Reply:
Yes, we are using its own style in carbon. This is carbon pagination but not a component. Carbon only provides guidance. We create our own pagination component base on the guidance.

Reply:
> Needs carbon prefix variable.
> Is the reasoning to use this specific carbon classname to inherit those styles?

Ok, we will still put prefix for these classnames. Working on it.

Reply:
Sounds good. Thanks